app/controllers/interviews_controller.rb

```diff
@@ -13,6 +13,7 @@ def new
 
   def show
     @interview = Interview.find(params[:id])
+    @questions = @interview.template.questions
```

Comment:
I added this relation in interview model and remove 'template' from @question.template.questions, but it does't work. We became error on summary page.